.github/workflows/ci.yml

```
@@ -0,0 +1,39 @@
+name: Continuous Integration
+
+on: [push, pull_request]
+
+jobs:
+  docker-build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Build Docker image
+      run: docker build .
+
+  Install-and-test:
```

```suggestion
  install-and-test:
```
for consistency with other names